x-pack/plugins/index_lifecycle_management/common/types/policies.ts

```diff
@@ -60,21 +60,13 @@ export interface SerializedActionWithAllocation {
   migrate?: MigrateAction;
 }
 
-export type SearchableSnapshotStorage = 'full_copy' | 'shared_cache';
-
 export interface SearchableSnapshotAction {
   snapshot_repository: string;
   /**
    * We do not configure this value in the UI as it is an advanced setting that will
    * not suit the vast majority of cases.
    */
   force_merge_index?: boolean;
-  /**
-   * This configuration lets the user create full or partial searchable snapshots.
-   * Full searchable snapshots store primary data locally and store replica data in the snapshot.
-   * Partial searchable snapshots store no data locally.
-   */
-  storage?: SearchableSnapshotStorage;
```

Comment:
pretty sure, this type was a leftover from searchable snapshot implementation that we ended up not needing